.gitignore

```diff
@@ -28,4 +28,5 @@
 
 # Ignore user's individual .env files
 .env
+.envrc
```

Comment:
Don't we normally keep this in the repo? It just tells direnv to use the .env file